pbsmetrics/prometheus/prometheus_test.go

```diff
@@ -407,6 +407,226 @@ func TestRequestTimeMetric(t *testing.T) {
 	}
 }
 
+func TestRecordStoredDataFetchTime(t *testing.T) {
+	tests := []struct {
+		description   string
+		giveDataType  pbsmetrics.StoredDataType
+		giveFetchType pbsmetrics.StoredDataFetchType
+		wantCount     uint64
+		wantSum       float64
+	}{
+		{
+			description:   "Update stored account histogram with all label",
+			giveDataType:  pbsmetrics.AccountDataType,
+			giveFetchType: pbsmetrics.FetchAll,
+			wantCount:     1,
```

Comment:
Seems like your count and sum are always the same values. Is there a reason they are part of the table structure?

Reply:
Nope, good point, removing.